bohye/hello-world/src/components/Menu.js

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+
+const Menu = () => {
+  return (
+    <div style={{ borderTop: "1px solid grey" }}>
+      <a
+        href="https://www.google.co.kr/?hl=ko"
+        target="_blank"
+        rel="noreferrer"
+      >
+        Google
+      </a>
+      <br />
+      <a href="https://www.naver.com/" target="_blank" rel="noreferrer">
+        Naver
+      </a>
```

Comment:
tag attribute indent는 둘 중 하나로 맞춰주는 것이 좋아보입니다.
```suggestion
      <a
        href="https://www.naver.com/"
        target="_blank"
        rel="noreferrer"
      >
        Naver
      </a>
```

Reply:
code style을 일관되게 작성하는 것은 프로그램 실행에는 영향을 주지 않으나, 매우 중요한 일입니다.   
작성된 코드의 일관성이 보장되지 않는다면, 동일한 기능에 대해서 다양한 예외를 발생시키게 되며 유지보수에 어려움을 주게 됩니다.